etc/generate-docs.sh

```diff
@@ -14,24 +14,11 @@ version=${1}
 # Ensure version is non-empty
 [ ! -z "${version}" ] || { echo "ERROR: Missing version string"; exit 1; }
 
-jazzy_args=(--clean
-            --author "Neal Beeken, Nellie Spektor, Patrick Freed, and Kaitlin Mahar" 
-            --readme "etc/docs-main.md" 
-            --author_url https://github.com/mongodb/swift-bson 
-            --github_url https://github.com/mongodb/swift-bson 
-            --theme fullwidth 
-            --documentation "Guides/*.md" 
-            --github-file-prefix https://github.com/mongodb/swift-bson/tree/v${version} 
-            --module-version "${version}" 
-            --swift-build-tool spm)
-
-modules=( BSON )
-
-for module in "${modules[@]}"; do
```

Comment:
I copied this from the driver where we have multiple modules, but it's not really necessary here. I moved all of the config options into the yaml file excluding those that are dynamically determined based on the version string (and also the output directory since the use of that is rather specific to how this script works.)